docs/hosting/configuration/configuration-examples/custom-certificate-authority.md

```diff
@@ -43,3 +43,9 @@ services:
             - 5678:5678
         image: docker.n8n.io/n8nio/n8n
 ```
+
+You should also give the right permissions to the imported certs. You can do this once the container is running:
+
+```bash
+docker exec --user 0 n8n chown -R 1000:1000 /opt/custom-certificates
```

Comment:
This will throw an error if someone runs this with
1. their n8n container named anything else besides `n8n`,
2. if they forgot to set a volume to mount to `/opt/custom-certificates`.
Do you think it might be worth noting that? or do you think these assumptions are implied?